source/channels/format-messages.rst

```diff
@@ -128,7 +122,11 @@ The message formatting toolbar offers the following formatting options:
 
 Review how your message formatting will look when the message is sent by selecting the **Show/Hide Preview** |preview-icon| icon. Return to your draft message by selecting the icon again.
 
-You can hide the formatting options by selecting the **Show/Hide Formatting** |hide-formatting-icon| icon. Select the icon again to show the formatting options.
+.. tip::
+
+  - Hide the formatting options by selecting the **Show/Hide Formatting** |hide-formatting-icon| icon. Select the icon again to show the formatting options. 
+  - Disable the formatting options by selecting the **Gear** |settings-icon| icon, going to **Settings > Advanced > Enable Post Formatting**, and setting this option to **Off**. 
```

Comment:
0/5, but I don't think we need to document this here since it's not something we expect people to leave on for long periods of times

Reply:
Docs has received customer feedback asking for details on how individual users can disable the new formatting options toolbar in their Mattermost clients. Is that what the Settings > Advanced > Enable Post Formatting offers? Or is to doing something else?

Reply:
No, it's for turning off Markdown rendering in posts. It was added originally to debug some very difficult to work around issues we used to have when Markdown was newer

Reply:
Thanks, @hmhealey! I'll update this docs PR accordingly.